src/sentry/utils/sentry_apps/webhooks.py

```diff
@@ -18,6 +18,7 @@
 
 
 WEBHOOK_TIMEOUT = 5
+TIMEOUT_STATUS_CODE = 0
```

Comment:
HTTP Timeout is 408

Reply:
I know, I actually don't know why this choice was made previously but I'm just making a constant for a bunch of places we used 0 as timeout.